### PR TITLE
Fix nuclei injection

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
Hi, I added for convenience the clang format of the O2, I can remove it of course and keep it locally if it is not needed

**Comment by njacazio** [comment]
@rutuparnarath

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,56 @@
+BasedOnStyle: Google
```
Comment (sawenzel) [inline .clang-format:1]: Please don't commit this file.

Comment (njacazio) [reply .clang-format:1]: It's ok, that's why I asked ;)

**Review by gconesab — APPROVED** [review]

File: .clang-format (hunk 1 above)
Comment (chiarazampolli) [reply .clang-format]: I agree.

Comment (njacazio) [reply .clang-format]: It should be already gone, no?

**Pull request merged**